ui/appui-react/src/appui-react/layout/widget/Tab.tsx

```diff
@@ -159,7 +154,6 @@ function WidgetTabComponent(props: WidgetTabProps) {
         className={className}
         ref={refs}
         role="tab"
-        style={props.style}
         tabIndex={0}
```

Comment:
Not necessarily related to this PR, but the whole tab pattern is also broken. `aria-selected` is missing so every tab is announced as "selected".

Recommend testing the entire experience with a screen reader.

Reply:
Good point, but I've not touched the tabs part yet (still using custom implementation), let's keep as is for now.